From 8c5348c0aa163c72d9976f987ea9de28d841c9a8 Mon Sep 17 00:00:00 2001
From: Jack Lo <jack.lo@technexion.com>
Date: Wed, 4 Mar 2026 10:56:02 +0800
Subject: [PATCH] arm64: dts: imx95-edm: disable in-kernel bluetooth binding
 for FCC testing

In FCC/Labtool test mode, the bluetooth chip on lpuart6 needs to be
controlled by the NXP labtool utility rather than the standard kernel
hci_uart driver. Removing the DT node prevents the kernel from
automatically binding the driver to the UART port.

Upstream-Status: Inappropriate [Configuration]

Signed-off-by: Jack Lo <jacklo@technexion.com>
---
 arch/arm64/boot/dts/freescale/imx95-edm.dtsi | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/imx95-edm.dtsi b/arch/arm64/boot/dts/freescale/imx95-edm.dtsi
index dc4969cd1..19a780b97 100644
--- a/arch/arm64/boot/dts/freescale/imx95-edm.dtsi
+++ b/arch/arm64/boot/dts/freescale/imx95-edm.dtsi
@@ -341,11 +341,6 @@ &lpuart6 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_uart6>;
 	status = "okay";
-
-	bluetooth {
-		compatible = "nxp,88w8997-bt";
-		fw-init-baudrate = <3000000>;
-	};
 };

 &micfil {
--
2.43.0

